templates/disk-uploader/readmes/README.md

```diff
@@ -0,0 +1,57 @@
+# KubeVirt Disk Uploader Task
```

Comment:
use this format of README https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/templates/create-vm-from-manifest/readmes
The Params, TaskRuns, examples will be autogenerated via the ansible playbook

Reply:
Thanks!